app.js

```diff
@@ -287,12 +317,26 @@ router.post('/account/sendRecoveryMessage', async ctx => {
     }
 
     const account = await models.Account.findOne({ where: { accountId } });
-    const [recoveryMethod] = await account.getRecoveryMethods({ where: {
-        kind: method.kind,
-        detail: method.detail
-    }});
 
-    await recoveryMethod.update({ publicKey, securityCode: null });
+    if (isNew) {
+        // remove all recovery methods
```

Comment:
nit: makes sense to explain that these might be coming from other users, etc

Reply:
clarified in upcoming commit